nvflare/fuel/f3/streaming/blob_streamer.py

```diff
@@ -44,64 +44,72 @@ def read(self, chunk_size: int) -> BytesAlike:
         return buf
 
 
+class BlobTask:
+    def __init__(self, future: StreamFuture, stream: Stream):
+        self.future = future
+        self.stream = stream
+        self.size = stream.get_size()
+        self.pre_allocated = self.size > 0
+
+        if self.pre_allocated:
+            self.buffer = wrap_view(bytearray(self.size))
+        else:
+            self.buffer = FastBuffer()
```

Comment:
not related to this PR, @nvidianz do we still need `FastBuffer()` ?

Reply:
Yes. It does the job for now. The bytearray() might work but I just haven't got time to do the test.